PKGBUILD

```diff
@@ -0,0 +1,20 @@
+# Maintainer: Julius de Jeu <julius@voidcorp.nl>
+pkgname=cadmium
+pkgver=0.1.0
+pkgrel=1
+pkgdesc="A simple Display Manager"
+makedepends=('rust' 'cargo')
```

Comment:
You can't actually install rust without cargo, and vice versa. So only one of them is needed, `cargo` seems to be the convention looking at other rust packages.

Reply:
I used the base `PKGBUILD` generated by the [cargo-pkgbuild](https://crates.io/crates/cargo-pkgbuild) crate, which specified both, I'll change this to just cargo. Makes more sense since depending on rust means that you cant use it with nightly only installed.